README.md

```diff
@@ -11,24 +11,26 @@ protected void Warning() {
 ```
 
 ## The most important thing
-I trust and expect my teams to do good work without me digging into the details, but I can and will dig all the way to bedrock when you or I need to.
+I trust and expect my teams to do good work without me digging into the details, but I can and will dig when you or I need to. I expect you to be the experts.
 
 ## What you can expect from me
 
 ### My working time
-I tend to work from 9:30-5:30 Eastern. I check slack and email earlier in the morning on my phone but it is rare that I will respond then. I endeavor to log out at 5:30 every day to be with my family and will not check slack or email after that time. For emergencies you can text me.
+I tend to work from 9:30-5:30 Eastern. I check slack and email earlier in the morning on my phone but it is rare that I will respond then and I definitely don't expect responses if I do. I log out at 5:30 every day to be with my family and will not check slack or email after that time. For emergencies you can text me. Because I don't check slack or email outside of my own decision to be working, you should feel totally safe to send me something at any time and I will read it when I'm back online.
```

Comment:
The wording on the second sentence is strange. Maybe:

```suggestion
I tend to work from 9:30-5:30 Eastern. I check slack and email earlier in the morning on my phone but it is rare that I will respond then and I definitely don't expect responses if I do. I log out at 5:30 every day to be with my family and will not check slack or email after that time. For emergencies you can text me. Feel free to send me a slack or email at any time: off-hours I will not get notified. Just don't expect a response until I'm back online.
```

Reply:
I agree this is clearer, though it doesn't feel like my tone of voice, so I'll tweak it.